react-view/src/pages/MultiWidgetPage/index.tsx

```
@@ -0,0 +1,90 @@
+
+  let Component: React.FunctionComponent<any>;
+  return (
+    <div>
+      {data.response.slots.map((datum, index) => {
+        let widgetData;
+        switch (datum.type) {
+          case "PRODUCT_INFO":
+            widgetData = datum.data as ProductInfoWidgetDataModel;
+            Component = ProductInfo;
+            break; 
+          case "PRODUCT_INFO":
```

Biome (1.9.4)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)